data/brands/shop/chemist.json

```diff
@@ -10,6 +10,18 @@
     }
   },
   "items": [
+     {
+      "displayName": "Caddy's",
+      "locationSet": {"include": ["it"]},
+      "matchTags": ["shop/convenience", "shop/houseware", "shop/cosmetics"],
```

Comment:
too much， don't need to include those matchTags,
you can refer to the matchgroups and only add a matchTag that don't in the matchgroup